src/ApplicationLogs/LogReader.cs

```diff
@@ -90,14 +90,19 @@ public static JObject TxLogToJson(Blockchain.ApplicationExecuted appExec)
             trigger["vmstate"] = appExec.VMState;
             trigger["exception"] = GetExceptionMessage(appExec.Exception);
             trigger["gasconsumed"] = appExec.GasConsumed.ToString();
-            try
+            var stack = new JArray();
+            foreach (var item in appExec.Stack)
             {
-                trigger["stack"] = appExec.Stack.Select(q => q.ToJson(Settings.Default.MaxStackSize)).ToArray();
-            }
-            catch (Exception ex)
-            {
-                trigger["exception"] = ex.Message;
+                try
+                {
+                    stack.Add(item.ToJson());
```

Comment:
The old one had `Settings.Default.MaxStackSize` parameter, but I'm not sure it affects anything here.